src/app/component/core/header-new/header-new.component.ts

```diff
@@ -41,16 +42,18 @@ export class HeaderNewComponent implements OnInit {
               private userService: UserService,
               private achievementService: AchievementService,
               private habitStatisticService: HabitStatisticService,
-              private languageService: LanguageService) {
+              private languageService: LanguageService,
+              private  userOwnAuthService: UserOwnAuthService) {
```

Comment:
Please remove 1 space between access modificator and the variable name

Reply:
fix